packages/stylelint-plugin/README.md

```diff
@@ -0,0 +1,77 @@
+# Kaizen Stylelint Plugin
+
+## Tl;dr
```

Comment:
Nitpick: I know _most_ people know what TL;DR is by now, but I feel "getting started" is just as good:

```suggestion
## Getting Started
```